src/test/data/script_tests.json

```diff
@@ -20,18 +20,20 @@
 ["0x02 0x0100", "", "P2SH,STRICTENC", "OK", "all bytes are significant, not only the last one"],
 ["0x09 0x000000000000000010", "", "P2SH,STRICTENC", "OK", "equals zero when cast to Int64"],
 
-["0x01 0x0b", "11 EQUAL", "P2SH,STRICTENC", "OK", "push 1 byte"],
+["0x5b", "11 EQUAL", "P2SH,STRICTENC", "OK", "push 1 byte"],
 ["0x02 0x417a", "'Az' EQUAL", "P2SH,STRICTENC", "OK"],
 ["0x4b 0x417a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a7a",
  "'Azzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz' EQUAL", "P2SH,STRICTENC", "OK", "push 75 bytes"],
 
-["PUSHDATA1 0x01 0x07","7 EQUAL", "P2SH,STRICTENC", "OK"],
-["PUSHDATA2 0x0100 0x08","8 EQUAL", "P2SH,STRICTENC", "OK"],
-["PUSHDATA4 0x01000000 0x09","9 EQUAL", "P2SH,STRICTENC", "OK"],
+["PUSHDATA1 0x4b 0x000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+ "1", "P2SH,STRICTENC", "MINIMALDATA"],
+["PUSHDATA1 0x4c 0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+ "SIZE 0x014c EQUAL", "P2SH,STRICTENC", "OK"],
+["PUSHDATA2 0xff00 0x000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+ "1", "P2SH,STRICTENC", "MINIMALDATA"],
+["PUSHDATA2 0x0001 0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+ "SIZE 0x020001 EQUAL", "P2SH,STRICTENC", "OK"],
 
-["PUSHDATA1 0x00","0 EQUAL", "P2SH,STRICTENC", "OK"],
-["PUSHDATA2 0x0000","0 EQUAL", "P2SH,STRICTENC", "OK"],
-["PUSHDATA4 0x00000000","0 EQUAL", "P2SH,STRICTENC", "OK"],
```

Comment:
For context; I removed these non-minimal pushes and replaced them with equivalent minimal pushes (or test for `MINIMALDATA` just at the boundary of becoming invalid)